control-plane/config/rbac/gatewaypolicy_editor_role.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
Hey John, just for cleanness sake, can you drop these files? Not sure what part of the generation step is adding them, but we generally don't commit any of these to our repos since they're just superfluous.